cmd/datum-authorization-webhook/app/internal/iam/core_control_plane_authorizer.go

```diff
@@ -0,0 +1,56 @@
+package iam
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+
+	"buf.build/gen/go/datum-cloud/iam/grpc/go/datum/iam/v1alpha/iamv1alphagrpc"
+	iampb "buf.build/gen/go/datum-cloud/iam/protocolbuffers/go/datum/iam/v1alpha"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/trace"
+	"k8s.io/apiserver/pkg/authorization/authorizer"
+)
+
+var _ authorizer.Authorizer = &CoreControlPlaneAuthorizer{}
+
+type CoreControlPlaneAuthorizer struct {
+	IAMClient iamv1alphagrpc.AccessCheckClient
+}
+
+// Authorize implements authorizer.Authorizer.
+func (o *CoreControlPlaneAuthorizer) Authorize(ctx context.Context, attributes authorizer.Attributes) (authorizer.Decision, string, error) {
+	req := &iampb.CheckAccessRequest{
+		Resource:   fmt.Sprintf("resourcemanager.datumapis.com/%s/%s", attributes.GetResource(), attributes.GetName()),
+		Subject:    "user:" + attributes.GetUser().GetName(),
+		Permission: attributes.GetVerb(),
+	}
+	ctx, span := otel.Tracer("go.datum.net/k8s-authz-webhook").Start(ctx, "datum.k8s-authz-webhook.global.Authorize", trace.WithAttributes(
```

Comment:
Do you want to tweak the `go.datum.net/k8s-authz-webhook` name now or later?

Reply:
I'll replace it now. Looks like this escaped the search I did earlier.

Reply:
Updated to `go.datum.net/datum/cmd/datum-authorization-webhook` to align on import path.